src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/Configuration.php

```diff
@@ -45,6 +45,7 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
                 ->booleanNode('intercept_redirects')->defaultFalse()->end()
+                ->booleanNode('enable_cors')->defaultFalse()->end()
```

Comment:
No downside, even if you're doing cross domain calls without exposing the related headers. However, you'll have security logs in your console.

This should be enabled if you're doing cross domain calls on Symfony apps that enabled cors and have been configured to expose these headers through CORS. Moreover, even if you do that, I'm not sure you would be able to display profiles (to be tested)